website/addons/s3/settings/defaults.py

```diff
@@ -4,6 +4,22 @@
 
 ALLOWED_ORIGIN = '*'
 
+BUCKET_LOCATIONS = {
```

Comment:
Just seeing this. Look at the latest push and see if it falls better in line with what you want (it follows the original pattern you encouraged)